libsodium-sys/README.md

```diff
@@ -0,0 +1,10 @@
+> libsodium-sys
+
+# Build output ENV Variables
+This is the build metadata for the crate. However, note that there is no such
+metadata if `SODIUM_LIB_DIR` is set.
+* `DEP_SODIUM_INCLUDE` is the directory which contains the `sodium.h` header.
```

Comment:
Is this var set always?

Reply:
yes it should be always set

Reply:
Even if you use system lib without headers?

Reply:
Unless `SODIUM_LIB_DIR` is set, as specified. That's because we cannot be sure that there are installed headers on the machine.

Reply:
Weird, just saw your second reply. Yes indeed if pkg_config or vcpkg don't find headers this won't be set. I'll fix that.